src/main/java/gregtech/api/metatileentity/MetaTileEntity.java

```diff
@@ -1174,9 +1174,17 @@ public static void clearInventory(NonNullList<ItemStack> itemBuffer, IItemHandle
         }
     }
 
+    /**
+     * Deprecated, use {@link MetaTileEntity#onPlacement()} instead
+     */
+    @Deprecated
     public void onAttached(Object... data) {
```

Comment:
There are still usages of this in GTCEu which you may need to investigate and convert to onPlacement(), or remove.